### PR TITLE
support core space fetcher and claimer.

### PR DESCRIPTION
**Pull request by Tigo9527**

### CODE REVIEW

File: cmd/addChain.go [hunk 1]
```diff
@@ -12,18 +12,18 @@ import (
 var addChainCmd = &cobra.Command{
 	Use:   "addChain",
 	Short: "Add a chain",
-	Long:  `Add a chain, args: chainId name rpc`,
-	Args:  cobra.ExactArgs(3),
+	Long:  `Add a chain, args: chainId name rpc <evm|cfx>`,
```
Comment (wanliqun) [inline cmd/addChain.go:15]: Recommend to use flag instead of argument to specify blockchain network

File: infra/blockchain/cfxFetcher.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+func GetCfxClient() *sdk.Client {
+	return nil
+}
+
+func NewCfxFetcher(chain int64, vaultAddr string, client *sdk.Client) (*CfxFetcher, error) {
+	filter, err := tokens.NewTokenVaultFilterer(cfxaddress.MustNewFromHex(vaultAddr, uint32(chain)), client)
+	if err != nil {
+		logrus.WithError(err).Error("NewTokenVaultFilterer by cfx fail")
```
Comment (wanliqun) [inline infra/blockchain/cfxFetcher.go:30]: Better to just return the error and let the business caller decide whether to log this error. To return more detailed error message, maybe you can use `errors.WithMessage(err, "detailed error message")`

File: infra/blockchain/cfxFetcher.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		return nil, err
+	}
+	fetcher := CfxFetcher{
+		Client: *client, filter: filter, vaultAddr: vaultAddr,
+	}
+	fetcher.filterOpts = &bind.FilterOpts{
+		Start: nil,
+		End:   nil,
+	}
+	return &fetcher, nil
```
Comment (wanliqun) [inline infra/blockchain/cfxFetcher.go:40]: I think you can just return quickly:
```
 CfxFetcher{
		Client: *client, filter: filter, vaultAddr: vaultAddr, filterOpts: &bind.FilterOpts{},
	}
```

File: infra/blockchain/cfxOperator.go [hunk 4]
```diff
@@ -121,6 +121,10 @@ func Cross721(client sdk.Client,
 	logrus.Info("claim by tx succeeded, hash ", claimTxHash)
 }
 
+func CfxClaim() {
+
+}
```
Comment (wanliqun) [inline infra/blockchain/cfxOperator.go:126]: // TODO?

Comment (Tigo9527) [reply infra/blockchain/cfxOperator.go]: Removed

File: infra/blockchain/cfxFetcher.go (hunk 2 above)
Comment (Tigo9527) [reply infra/blockchain/cfxFetcher.go]: Fixed.

**Review by wanliqun — APPROVED** [review]

**Pull request merged**